src/assets/buttons.scss

```diff
@@ -22,6 +22,11 @@
 
 @import 'variables';
 
+.button-vue--vue-default {
+	background-color: var(--color-background-dark);
+	color: var(--color-main-text);
+}
```

Comment:
This looks wrong. If we need a mode of the button that does not exist yet, we should fix it in the vue library

Reply:
Should I wait the updated Vue library? ;)

Reply:
No need, I would just use the secondary button here. Whenever you see that gray-ish button, you should replace with the current type="secondary" from the button component. 

<img width="352" alt="Screenshot 2022-03-02 at 18 03 48" src="https://user-images.githubusercontent.com/26852655/156411367-78bc2a0c-bfe7-48ea-9feb-aa58faba8a10.png">

Reply:
Ok